proxmox/README.md

```diff
@@ -0,0 +1,60 @@
+# Agent Check: Proxmox
+
+## Overview
+
+This check monitors [Proxmox][1] through the Datadog Agent. 
+
+Include a high level overview of what this integration does:
+- What does your product do (in 1-2 sentences)?
+- What value will customers get from this integration, and why is it valuable to them?
+- What specific data will your integration monitor, and what's the value of that data?
```

Comment:
Hey @aureleoules Could you try to put a description together here and answer the questions outlined?